src/mockchunks.jl

```diff
@@ -27,7 +27,15 @@ The chunks argument can take one of the following forms:
 - a [`DiskArrays.GridChunks`](@ref) object
 - a tuple specifying the chunk size along each dimension, like `(10, 10, 1)` for a 3-D array
 """
-mockchunks(data::AbstractArray, chunks) = MockChunkedDiskArray(data, GridChunks(size(data), chunks))
+function mockchunks(data::AbstractArray, chunks)
```

Comment:
Shouldn't we be doing that via Dispatch on chunks?

Reply:
I want to leave room to dispatch on the first argument, and not have ambiguities if the second argument is typed as well.  DD does override `mockchunks(::AbstractDimArray, chunks)`, which would introduce an ambiguity if you ever did `mockchunks(::AbstractDimArray, ::GridChunks)` if we did this by dispatch.

It should be equally fast but maybe I can add a specialization to make sure.

```suggestion
function mockchunks(data::AbstractArray, chunks::C) where C
```